test/solidity_array_structs.js

```diff
@@ -0,0 +1,75 @@
+const Web3 = require("web3");
+const assert = require("assert");
+const Ganache = require("../index");
+const path = require("path");
+const compileAndDeploy = require("./helpers/contracts").compileAndDeploy;
+
+const mnemonic = "candy maple cake sugar pudding cream honey rich smooth crumble sweet treat";
+
+function setUp(options = { mnemonic }, contractName = "ArrayOfStructs") {
+  const context = {
+    options: options,
+    provider: null,
+    web3: null,
+    accounts: [],
+    contractArtifact: {},
+    instance: null
+  };
+
+  before("setup web3", async function() {
+    // eslint-disable-next-line new-cap
+    context.provider = new Ganache.provider(context.options);
```

Comment:
I don't think we should be disabling the linter if it can be avoided.  On line 21, you can remove the `new` keyword and that should fix the linter error